### DISCUSSION TITLE
Theme design of w2ui.

### DISCUSSION THREAD

**Discussion by vinhpt2** [vitmalina/w2ui#2590] [General]
Hi, I am intent to make other theme for w2ui 2.0. So do any one have figma design of w2ui 2.0 to share. It will save me many time. I promise to contribute back any theme I make in future Thanks!

**Comment by mansurdegirmenci** [comment]
It wouldn't be bad if you work on a dark theme :)

**Reply by vinhpt2** [reply]
I still need Figma file for save time first :)
